server/scripts/README.md

```diff
@@ -0,0 +1,29 @@
+# Server Scripts
+
+This directory contains backend scripts that do things outside the normal user flow of Waterpark.
+
+## Getting Started
+
+Unlike the rest of this repository, the scripts are written in Python, so special setup is needed:
+
+1. Install [Python 3](https://www.python.org/downloads/).
```

Comment:
super nit: can we have platform specific instructions on our readme?

for windows, navigate them to the python installers via website
for mac users, installing with brew is always the go to. `brew install python`

Reply:
good point, I'll link them to the universal installation guide then

https://realpython.com/installing-python/